### ISSUE TITLE
pipenv does not work with '(' or ')' characters.

### ISSUE THREAD

**Issue by DirtySoc** [pypa/pipenv#3168]
### Issue description

If you are in a directory with spaces and '(' or ')' in the name then pipenv fails to work.

### Expected result

pipenv to create a venv, activate it, and install other modules in it.

### Actual result

pipenv created a virtualenv but could not activate it. The resulting error shows that the command path was truncated at the first '(' in the directory name.

### Steps to replicate

On a windows OS:
1. py -m pipenv install pyperclip

-------------------------------------------------------------------------------

<details><summary>$ pipenv --support</summary>

Pipenv version: `'2018.10.13'`

Pipenv location: `'C:\\Users\\derek\\AppData\\Local\\Programs\\Python\\Python37\\lib\\site-packages\\pipenv'`

Python location: `'C:\\Users\\derek\\AppData\\Local\\Programs\\Python\\Python37\\python.exe'`

Python installations found:

  - `5.2.0`: `C:\Users\derek\Anaconda3\python.exe`
  - `5.2.0`: `C:\Program Files (x86)\Microsoft Visual Studio\Shared\Anaconda3_64\python.exe`
  - `3.7.0`: `C:\Users\derek\AppData\Local\Programs\Python\Python37\python.exe`
  - `3.6.6`: `C:\Users\derek\AppData\Local\Programs\Python\Python36\python.exe`
  - `2.7`: `C:\Python27\python.exe`

PEP 508 Information:

```
{'implementation_name': 'cpython',
 'implementation_version': '3.7.0',
 'os_name': 'nt',
 'platform_machine': 'AMD64',
 'platform_python_implementation': 'CPython',
 'platform_release': '10',
 'platform_system': 'Windows',
 'platform_version': '10.0.17763',
 'python_full_version': '3.7.0',
 'python_version': '3.7',
 'sys_platform': 'win32'}
```

System environment variables:

  - `ALLUSERSPROFILE`
  - `APPDATA`
  - `CHOCOLATEYINSTALL`
  - `CHOCOLATEYLASTPATHUPDATE`
  - `COMMONPROGRAMFILES`
  - `COMMONPROGRAMFILES(X86)`
  - `COMMONPROGRAMW6432`
  - `COMPUTERNAME`
  - `COMSPEC`
  - `CUDA_PATH`
  - `CUDA_PATH_V10_0`
  - `DRIVERDATA`
  - `HOMEDRIVE`
  - `HOMEPATH`
  - `KMP_DUPLICATE_LIB_OK`
  - `LOCALAPPDATA`
  - `LOGONSERVER`
  - `MKL_SERIAL`
  - `MSMPI_BIN`
  - `NIDAQMXSWITCHDIR`
  - `NIIVIPATH`
  - `NUMBER_OF_PROCESSORS`
  - `NVCUDASAMPLES10_0_ROOT`
  - `NVCUDASAMPLES_ROOT`
  - `NVTOOLSEXT_PATH`
  - `ONEDRIVE`
  - `OS`
  - `PATH`
  - `PATHEXT`
  - `PROCESSOR_ARCHITECTURE`
  - `PROCESSOR_IDENTIFIER`
  - `PROCESSOR_LEVEL`
  - `PROCESSOR_REVISION`
  - `PROGRAMDATA`
  - `PROGRAMFILES`
  - `PROGRAMFILES(X86)`
  - `PROGRAMW6432`
  - `PSMODULEPATH`
  - `PUBLIC`
  - `QSYS_ROOTDIR`
  - `QUARTUS_ROOTDIR`
  - `SESSIONNAME`
  - `SOPC_KIT_NIOS2`
  - `SYSTEMDRIVE`
  - `SYSTEMROOT`
  - `TEMP`
  - `TESTSTAND`
  - `TESTSTANDAPPDATA`
  - `TESTSTANDPUBLIC`
  - `TMP`
  - `USERDOMAIN`
  - `USERDOMAIN_ROAMINGPROFILE`
  - `USERNAME`
  - `USERPROFILE`
  - `VBOX_MSI_INSTALL_PATH`
  - `VXIPNPPATH`
  - `WINDIR`
  - `PYTHONDONTWRITEBYTECODE`
  - `PIP_SHIMS_BASE_MODULE`
  - `PIP_PYTHON_PATH`

Pipenv?specific environment variables:

Debug?specific environment variables:

  - `PATH`: `C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v10.0\bin;C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v10.0\libnvvp;C:\Program Files\Microsoft MPI\Bin\;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\WINDOWS\System32\OpenSSH\;C:\Program Files\PuTTY\;C:\Program Files (x86)\IVI Foundation\IVI\bin;C:\Program Files (x86)\IVI Foundation\VISA\WinNT\Bin\;C:\PROGRA~2\IVIFOU~1\VISA\WinNT\Bin;C:\Program Files (x86)\IVI Foundation\VISA\WinNT\Bin;C:\Program Files\dotnet\;C:\Program Files\Microsoft SQL Server\130\Tools\Binn\;C:\Program Files\Intel\WiFi\bin\;C:\Program Files\Common Files\Intel\WirelessCommon\;C:\Program Files\TortoiseSVN\bin;C:\Program Files (x86)\Plantronics\Spokes3G\;C:\Program Files (x86)\Yarn\bin\;C:\ProgramData\chocolatey\bin;C:\Program Files\nodejs\;C:\Program Files (x86)\NVIDIA Corporation\PhysX\Common;C:\Program Files\Git\cmd;C:\Users\derek\AppData\Local\Microsoft\WindowsApps;C:\Users\derek\AppData\Local\Programs\Microsoft VS Code\bin;C:\intelFPGA_lite\18.1\modelsim_ase\win32aloem;C:\Program Files\Intel\WiFi\bin\;C:\Program Files\Common Files\Intel\WirelessCommon\;C:\Users\derek\AppData\Local\Yarn\bin;C:\Users\derek\AppData\Roaming\npm;C:\Program Files\nodejs\;`

---------------------------

Contents of `Pipfile` ('C:\\Users\\USER\\Documents\\repos\\Report Generator (Python)\\Pipfile'):

```toml
[[source]]
url = "https://pypi.org/simple"
verify_ssl = true
name = "pypi"

[packages]

[dev-packages]

[requires]
python_version = "3.7"

```

</details>

**Comment by techalchemy** [comment]
Thanks @frostming, we also have issues around non-ascii paths related to prettytoml... would love to get rid of that dependency but the toml landscape doesn't handle comments with any kind of consistency :/

**Comment by frostming** [comment]
@techalchemy Yes, [atoml](https://pypi/p/atoml) doens't support this either. It is a non-trivial change and may involve rewriting the parsing logic. But I will try to fix the non-ascii support for prettytoml in the next comming patch.

**Comment by frostming** [comment]
Closed via #3186
